### PR TITLE
Add lie algebra adjoint operators

### PR DESCRIPTION
**Pull request by JohnKJung**

### CODE REVIEW

File: pymlg/jax/se23.py [hunk 1]
```diff
@@ -194,6 +194,17 @@ def adjoint(X):
         Ad = Ad.at[6:9, 6:9].set(C)
         return Ad
 
+    @staticmethod
+    @jit
+    def adjoint_algebra(Xi):
+        A = jnp.zeros((9, 9))
+        A[0:3, 0:3] = Xi[0:3, 0:3]
```
Comment (CharlesCossette) [inline pymlg/jax/se23.py:201]: unfortunately jax arrays dont allow this kind of indexing, you have to do something more similar to the above

**Review by CharlesCossette — CHANGES_REQUESTED** [review]
It would be awesome if you added a single test in the `standard_tests.py` file that makes sure these new functions work correctly. If you do that, it will be applied to ALL the groups in ALL the implementations. If I recall correctly, theres a general identity that the little adjoint should satisfy. This is the perfect thing to test for. 

https://github.com/decargroup/pymlg/blob/main/tests/standard_tests.py#L142

**Review by CharlesCossette — APPROVED** [review]
awesome!

**Comment by CharlesCossette** [comment]
Strange that some `SL3` tests are suddenly failure now. it looks completely unrelated to your change.

**Comment by JohnKJung** [comment]
That is strange...

**Comment by CharlesCossette** [comment]
Fixed it. It was an issue with using random numbers in the unit tests.

**Pull request merged**